README.md

```diff
@@ -24,6 +24,46 @@ the following features in mind:
 * Be well-tested and robust to the point that it can be used in large
   projects without exposing unfortunate, disappointing bugs here and there.
 
+## Building
+
+Ormolu builds with `cabal-install` and `ghc-8.4`. For instance:
+
+```bash
+$ nix-shell -p haskell.compiler.ghc844
+<inside shell>
+$ cabal new-update
+$ cabal new-build
+```
+
+Alternatively, `stack` could be used with a `stack.yaml` file as follows.
+```bash
+$ cat stack.yaml
+resolver: lts-12.26
+packages:
+- '.'
+extra-deps:
+- path-io-1.4.2
+```
+
+## Usage
+
+The following will replace the contents of a Haskell module with the
+formatted output.
+```bash
+$ ormolu Module.hs
```

Comment:
This will just print formatted `Module.hs` to stdout. You need `--mode inplace` to  overwrite it.

Reply:
Done.